add-cache-control-header/index.js

```diff
@@ -1,12 +1,12 @@
-function handler(event) {
-    var response = event.response;
-    var headers = response.headers;
-
+async function handler(event) {
+    const response = event.response;
```

Comment:
I don't think these can be const since you are changing the values within them. I think using `var` is correct